packages/vm/src/runTx.ts

```diff
@@ -47,7 +47,7 @@ export interface RunTxOpts {
    */
   skipNonce?: boolean
   /**
-   * If true, skips the balance check
+   * Skip balance checks if true. Adds transaction value to balance to ensure execution doesn't fail.
```

Comment:
nit, this is different than runCall
```suggestion
   * Skip balance checks if true. Adds transaction cost to balance to ensure execution doesn't fail.
```